evap/evaluation/templates/base.html

```diff
@@ -97,13 +97,17 @@
     templateForm.insertAfter($('.select2form'));
 
     // put this into a function for later re-use
-    applySelect2 = function(elem) {
+    applySelect2 = function(elem, minimumInput = false) {
         elem.select2({
             language: "{{ LANGUAGE_CODE }}",
             placeholder: "{% trans 'Please select...' %}",
-            minimumResultsForSearch: 15,});
+            minimumResultsForSearch: 15,
+            minimumInputLength: minimumInput ? 3 : 0
+        });
     };
-    applySelect2($('select').not('.form-template select').not('[disabled]'));
+    $('select').not('.form-template select').not('[disabled]').each(function() {
+        applySelect2($(this), $(this).children('option').length > 50);
```

Comment:
since the parameter is set only here, i would inline that.

Reply:
like how?

Reply:
ah, i think i know what you mean.

Reply:
i'm not quite sure what you're asking, i was thinking something like `minimumInputLength: elem..children('option').length > 50 ? 3 : 0`

actually, if i understand correctly, without that you'd be introducing a bug. `applySelect2` is called e.g. when adding a form to the contribution formset on the course edit page, and the added forms would have a `minimumInputLength` of 0 while the initially visible forms would have set that to 3.

Reply:
does that still work when calling `applySelect2` with a single element like when adding a form?

Reply:
yes. e.g., on the course edit page i can add additional contributors and the selection field there still requires three characters to be typed before showing options.